go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/decred/dcrd/chaincfg v1.5.1
 	github.com/decred/dcrd/chaincfg/chainhash v1.0.1
 	github.com/decred/dcrd/connmgr v1.0.2
+	github.com/decred/dcrd/crypto/blake256 v1.1.0
```

Comment:
This is a distinct module from the upstream module, so it should start at `v1.0.0`.